### PR TITLE
feat: text (typography) component

### PR DESCRIPTION
**Pull request by ad2ien**
This PR implements a basic typography component.

Specified by the following issue : https://github.com/okp4/ui/issues/98

### CODE REVIEW

**Review by nperon — DISMISSED** [review]
Great!

**Review by ErikssonJoakim — APPROVED** [review]
Nice work!

**Review by nperon — DISMISSED** [review]

File: src/ui/styles/font.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@font-face {
+  font-family: secondary-light;
+  src: url('assets/fonts/NotoSansMono-Light.ttf');
+}
+@font-face {
+  font-family: secondary-medium;
+  src: url('assets/fonts/NotoSansMono-Medium.ttf');
+}
+@font-face {
+  font-family: secondary-bold;
```
Comment (amimart) [inline src/ui/styles/font.scss:22]: `secondary-block` & `secondary-black` are already defined above.

File: src/ui/atoms/text/Text.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames'
+import React from 'react'
+import type { DeepReadonly } from '../../../superTypes'
+import './text.scss'
+
+export type TTextProps = DeepReadonly<{
```
Comment (amimart) [inline src/ui/atoms/text/Text.tsx:6]: Based on the @ccamel [comment](https://github.com/okp4/ui/pull/81#discussion_r843176700) the convention dictate to not prefix types with `T`.

File: src/ui/atoms/text/Text.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  /**
+   * Prop used to switch out what's being ultimately rendered. By default, a `span` component.
```
Comment (amimart) [inline src/ui/atoms/text/Text.tsx:8]: Based on the @ccamel [comment](https://github.com/okp4/ui/pull/81#discussion_r843181124), maybe we need to remove the defaults notions here.

**Review by amimart — CHANGES_REQUESTED** [review]
Some comments based on @ccamel ones in another PR 😉

**Review by amimart — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]
That's a great job, thanks! ☺️

**Review by ccamel — APPROVED** [review]
Really nice! 🥰

**Pull request merged**